client/src/areas/party/home/PartyHome.tsx

```diff
@@ -11,7 +11,7 @@ export class PartyHome extends React.Component {
         <PartyLoginForm />
         <PartyExplanation />
         <LeftSideButtons>
-          <HomeButton />
+          <HomeButton switchToBaseTheme={true} />
```

Comment:
this should not be done like this, I will maybe tomorrow look into this myself :)